packages/shared/src/components/modals/PostToSquadModal.tsx

```diff
@@ -51,19 +46,22 @@ function PostToSquadModal({
     file: squad.image,
     handle: squad.handle,
     buttonText: 'Done',
-    url,
+    privateLink,
   });
 
   const onPostSuccess = async (squadPost?: Post) => {
+    const syncSubmission = !!form?.privateLink?.title || !!form?.post?.post;
+
     if (squadPost) onSharedSuccessfully?.(squadPost);
 
     displayToast(
-      isLink
+      !syncSubmission
```

Comment:
In what case can it be non sync submission?
Pretty sure we block anything that's not sync now right.

Reply:
If the title is not found from the API where we fetch the preview.